src/game.cpp

```diff
@@ -2851,6 +2850,16 @@ void Game::toggleCinematic(float *statustext_time)
 	statustext = msg[cinematic];
 }
 
+// Add WoW-style autorun by toggling continuous forward.
+void Game::toggleAutorun(float *statustext_time)
+{
+	static const wchar_t *msg[] = { L"autorun disabled", L"autorun enabled" };
+	bool autorun = !g_settings->getBool("continuous_forward");
+	g_settings->set("continuous_forward", bool_to_cstr(autorun));
+
+	*statustext_time = 0;
+	statustext = msg[autorun];
```

Comment:
The C++ Standard says it is guaranteed be 0 or 1 (1 for true). That said, your suggestion is probably easier to read (in this specific case) and the compiler will (probably) optimise it anyway.